[2024-07-30T10:28:18.539+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:28:18.587+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Load_songplays_fact_table scheduled__2018-11-29T00:00:00+00:00 [queued]>
[2024-07-30T10:28:18.610+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Load_songplays_fact_table scheduled__2018-11-29T00:00:00+00:00 [queued]>
[2024-07-30T10:28:18.611+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:28:18.888+0000] {taskinstance.py:2330} INFO - Executing <Task(LoadFactOperator): Load_songplays_fact_table> on 2018-11-29 00:00:00+00:00
[2024-07-30T10:28:18.903+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1376) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:28:18.906+0000] {standard_task_runner.py:63} INFO - Started process 1388 to run task
[2024-07-30T10:28:18.902+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Load_songplays_fact_table', 'scheduled__2018-11-29T00:00:00+00:00', '--job-id', '288', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpilfhn765']
[2024-07-30T10:28:18.907+0000] {standard_task_runner.py:91} INFO - Job 288: Subtask Load_songplays_fact_table
[2024-07-30T10:28:19.110+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Load_songplays_fact_table scheduled__2018-11-29T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:28:19.443+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Load_songplays_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2018-11-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-29T00:00:00+00:00'
[2024-07-30T10:28:19.445+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:28:19.486+0000] {load_fact_operator.py:24} INFO - Loading fact table songplays
[2024-07-30T10:28:19.528+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:28:19.559+0000] {sql.py:487} INFO - Running statement: 
        INSERT INTO songplays (playid, start_time, userid, level, songid, artistid, sessionid, location, user_agent)
        SELECT
            md5(events.sessionid::integer|| events.start_time::text) AS songplay_id,
            events.start_time, 
            events.userid, 
            events.level, 
            songs.song_id, 
            songs.artist_id, 
            events.sessionid, 
            events.location, 
            events.useragent
        FROM (SELECT TIMESTAMP 'epoch' + ts/1000 * interval '1 second' AS start_time, *
              FROM staging_events
              WHERE page='NextSong') events
        LEFT JOIN staging_songs songs
        ON events.song = songs.title
           AND events.artist = songs.artist_name
           AND events.length = songs.duration
        ON CONFLICT (playid) DO NOTHING;
    , parameters: None
[2024-07-30T10:28:19.703+0000] {sql.py:496} INFO - Rows affected: 319
[2024-07-30T10:28:19.880+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:28:19.911+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Load_songplays_fact_table, run_id=scheduled__2018-11-29T00:00:00+00:00, execution_date=20181129T000000, start_date=20240730T102818, end_date=20240730T102819
[2024-07-30T10:28:20.013+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:28:20.092+0000] {taskinstance.py:3503} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:28:20.177+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
